src/__tests__/transfers.test.ts

```diff
@@ -5,23 +5,25 @@ import { BigNumber } from "bignumber.js";
 import { expect } from "chai";
 
 import { erc20Interface } from "../erc20";
-import { fetchTokenList, TokenMap } from "../hooks/tokenList";
+import { fetchTokenList, MinimalTokenInfo, TokenMap } from "../hooks/token";
 import { Payment } from "../parser";
 import { testData } from "../test/util";
 import { buildTransfers } from "../transfers";
 import { toWei, fromWei, MAX_U256 } from "../utils";
 
 const dummySafeInfo = testData.dummySafeInfo;
 let tokenList: TokenMap;
-let listedTokens: string[];
-let listedToken: TokenInfo;
+let listedToken: MinimalTokenInfo;
 const receiver = testData.addresses.receiver1;
 
 describe("Build Transfers:", () => {
   beforeAll(async () => {
     tokenList = await fetchTokenList(dummySafeInfo.network);
-    listedTokens = Array.from(tokenList.keys());
-    listedToken = tokenList.get(listedTokens[0]);
+    let listedTokens = Array.from(tokenList.keys());
+    const tokenInfo = tokenList.get(listedTokens[0]);
+    if (typeof tokenInfo !== "undefined") {
+      listedToken = tokenInfo;
+    }
```

Comment:
What would happen if this weren't true? Shouldn't we instead make an assertion here?